indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUsedAction.java

```diff
@@ -53,9 +69,9 @@ public String getDataSource()
   }
 
   @JsonProperty
```

Comment:
MMs will have to be updated after overlords for this to work, but I think that's normal, so that's cool.